tests/test_satfunc.py

```diff
@@ -310,6 +310,129 @@ def test_wrong_columns():
     assert len(wrongdf["SW"].unique()) == 3
 
 
+def test_defaulted_values():
```

Comment:
This should probably be split into different tests.

Reply:
fixed.